secboot/secboot_tpm_test.go

```diff
@@ -494,6 +499,26 @@ func (s *secbootSuite) TestSealKey(c *C) {
 			mockEFI[0] = "/does/not/exist"
 		}
 
+		// create a snap file
+		foo := filepath.Join(tmpDir, "foo.snap")
```

Comment:
Would it be more realistic to call this "kernel.snap" and build it with SnapType: "kernel" below? Probably makes not differences but AIUI we only use this for kernels, yes?